defaults/main.yml

```diff
@@ -9,3 +9,5 @@ logging_outputs: []
 logging_custom_config_files: []
 
 logging_purge_confs: false
+
+use_python3: true
```

Comment:
not sure why this is needed

Reply:
This is a workaround for this issue.  (Please see the conversation attached to the area)
https://github.com/linux-system-roles/logging/pull/80/files#diff-985705db1379cc4bdac80eaa88789109R14-R22

Also please see my comment https://github.com/linux-system-roles/logging/pull/80#discussion_r364941142.  Not sure in which layer (ansible, molecule or the way how I use them...), but there's some confusion in switching the python version 2 to 3...

Reply:
@nhosoi where is the issue please? If I remove `use_python3` and the associated istallation of libselinux, the Travis CI build passes. To me this indicates that there has been no issue with Molecule in the first place...  #82